pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -744,8 +744,9 @@ func (p *provider) newVirtualMachine(c *Config, pc *providerconfigtypes.Config,
 					},
 					Domain: kubevirtv1.DomainSpec{
 						Devices: kubevirtv1.Devices{
-							Interfaces: []kubevirtv1.Interface{*defaultBridgeNetwork},
-							Disks:      getVMDisks(c),
+							Interfaces:                 []kubevirtv1.Interface{*defaultBridgeNetwork},
+							Disks:                      getVMDisks(c),
+							NetworkInterfaceMultiQueue: pointerToBool(true),
```

Comment:
you can use ptr utils:

```go
import "k8s.io/utils/ptr"
```

then,

```go
NetworkInterfaceMultiQueue: ptr.To(true)
```

Reply:
k8s.io/utils/ptr dependancy is already present. I have changed the NetworkInterfaceMultiQueue function to ptr.To(true)